PrimeroAdministrationandConfigurationGuide.md

```diff
@@ -294,6 +294,21 @@ Although not recommended, if for some reason a user needs to be deleted, follow
 
 To delete the username, click on the OK button. To cancel and keep the username, click on the CANCEL button. This action cannot be undone so proceed with caution.
 
+## Configuring Manager Dashboard
+
+The statistics that show up on the manager are configurable from the roles settings. To change the dashboard for a role, click on the **Settings** tab and click on **Roles**. Scroll down to the **Dashboards** section and here you can add or remove modules.
```

Comment:
Once again, not sure I would call these "modules."
Also, "that show up on the manager are . . ." should be "that show up on the manager dashboard are . . ."